test/index.js

```diff
@@ -0,0 +1,36 @@
+const {zip} = require('rxjs');
```

Comment:
something about "check the oldest supported `@polkadot/rpc-provider@2.7.1` is compatible", "we don't control the RPC details from @cennznet/api but require that they don't change to provide LTS for users"